src/lib.rs

```diff
@@ -30,15 +31,19 @@
 //! # fn main() {
 //! // Construct an `LruCache` of `<u8, String>`s, limited by key count
 //! let max_count = 10;
-//! let lru_cache = LruCache::<u8, String>::with_capacity(max_count);
+//! let _lru_cache = LruCache::<u8, String>::with_capacity(max_count);
 //!
 //! // Construct an `LruCache` of `<String, i64>`s, limited by expiry time
 //! let time_to_live = ::std::time::Duration::from_millis(100);
-//! let lru_cache = LruCache::<String, i64>::with_expiry_duration(time_to_live);
+//! let _lru_cache = LruCache::<String,
+//! i64>::with_expiry_duration(time_to_live);
```

Comment:
It seems to have broken every line in the file that was longer than 80 characters, even though the limit is 100.